README.md

```diff
@@ -90,17 +97,341 @@ for page transitions, and scrollable content within each page.
     + [Pagination Animation](#pagination-animation)
     + [Scrolling Animation](#scrolling-animation)
   * [Example Configuration](#example-configuration)
-- [Usage](#usage)
-  * [Example app](#example-app)
-  * [Playground app with imperative navigation](#playground-app-with-imperative-navigation)
-  * [Playground app with declarative navigation](#playground-app-with-declarative-navigation)
-  * [Coffee maker app for state management example](#coffee-maker-app-for-state-management-example)
-  * [CupertinoApp support](#cupertinoapp-support)
 - [Additional information](#additional-information)
 
-## Examples
 
-You can see how the package is used in the example apps included in the 
+## Usage
+
+This package includes four example projects, but if you're looking to get a quick feel for it, check out the basic starter app.
+
+> Besides the "Basic Starter App," be sure to clone the related example repository to get a feel for the fully functioning samples.
+
+### Basic Starter App
+
+Install the library to your project by running: 
+
+```bash
+flutter pub add wolt_modal_sheet
+```
+
+Afterwards, if you copy and paste following code to your `main.dart` file and run it, it will help you get started:
+
+```dart
+import 'package:flutter/material.dart';
+import 'package:wolt_modal_sheet/wolt_modal_sheet.dart';
+
+void main() {
+  runApp(
+    const WoltModalSheetApp(),
+  );
+}
+
+class WoltModalSheetApp extends StatelessWidget {
+  const WoltModalSheetApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: WoltModalSheetHomePage(),
+    );
+  }
+}
+
+class WoltModalSheetHomePage extends StatelessWidget {
+  const WoltModalSheetHomePage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Wolt Modal Bottom Sheet Sample'),
+      ),
+      floatingActionButton: FloatingActionButton.extended(
+        onPressed: () {
+          WoltModalSheet.show(
+            context: context,
+            pageListBuilder: (bottomSheetContext) => [
+              SliverWoltModalSheetPage(
+                mainContentSlivers: [
```

Comment:
we are just deprecating `mainContentSlivers`. Please refer to this PR: https://github.com/woltapp/wolt_modal_sheet/pull/286